yt_dlp/extractor/allstar.py

```diff
@@ -0,0 +1,259 @@
+        'info_dict': {
+            'id': '62b8bdfc9021052f7905882d',
+            'title': 'cherokee - Clips',
+        },
+        'playlist_mincount': 15
+    }, {
+        'url': 'https://allstar.gg/u/cherokee?game=730&view=Clips',
+        'info_dict': {
+            'id': '62b8bdfc9021052f7905882d',
+            'title': 'cherokee - Clips',
+        },
+        'playlist_mincount': 15
+    }, {
+        'url': 'https://allstar.gg/u/62b8bdfc9021052f7905882d?view=Montages',
+        'info_dict': {
+            'id': '62b8bdfc9021052f7905882d',
+            'title': 'cherokee - Montages',
+        },
+        'playlist_mincount': 4
+    }, {
+        'url': 'https://allstar.gg/profile?user=cherokee&view=Mobile Clips',
+        'info_dict': {
+            'id': '62b8bdfc9021052f7905882d',
+            'title': 'cherokee - Mobile Clips',
+        },
+        'playlist_mincount': 1
+    }]
+
+    _PAGE_SIZE = 10
+
+    @staticmethod
+    def _set_webpage_url(info_dict):
+        video_id = info_dict.get('id')
+        video_url = info_dict.get('url')
+
+        if video_url is None or video_id is None:
+            return info_dict
+
+        base_name = 'clip' if '/clips/' in video_url else 'montage'
+        info_dict['webpage_url'] = f'https://allstar.gg/{base_name}?{base_name}={video_id}'
+        info_dict['webpage_url_basename'] = base_name
+
+        return info_dict
+
+    def _get_page(self, user_id, display_id, game, query, page_num):
+        page_num += 1
+
+        for video_data in self._send_query(
+                query, {
+                    'user': user_id,
+                    'page': page_num,
+                    'game': game,
+                }, ('data', 'videos', 'data'), display_id, f'Downloading page {page_num}'):
+            yield self._set_webpage_url(self._parse_video_data(video_data))
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        profile_data = self._download_json(
+                urljoin('https://api.allstar.gg/v1/users/profile/', display_id),
+                display_id)
+        user_id = traverse_obj(profile_data, ('data', ('_id'), {str_or_none}))
+
+        if user_id is None:
+            raise ExtractorError('Can not extract the user_id')
+
+        username = traverse_obj(profile_data, ('data', 'profile', ('username'), {str_or_none}))
+        url_query = parse_qs(url)
+        game = traverse_obj(url_query, ('game', 0, {int_or_none}))
+        view = traverse_obj(url_query, ('view', 0, {str_or_none}), default='Clips')
+        query_id = view
+
+        if query_id not in ('Clips', 'Montages', 'Mobile Clips'):
+            raise UnsupportedError(url)
+
+        return self.playlist_result(
+            OnDemandPagedList(
+                functools.partial(
+                    self._get_page, user_id, display_id, game, _QUERIES.get(query_id)), self._PAGE_SIZE),
+            user_id, f'{username or display_id} - {view}')
```

Comment:
With the change from above of renaming `view` to `query_id`
```suggestion
            user_id, f'{username or display_id} - {query_id}')
```